org.vikdevelop.VCalculator.json

```diff
@@ -0,0 +1,36 @@
+ "app-id" : "org.vikdevelop.VCalculator",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "vcalculator",
+    "finish-args" : [
+        "--share=network",
```

Comment:
do you really need network?

Reply:
I updated file com.github.vikdevelop.VCalculator